index.bs

```diff
@@ -1223,12 +1223,21 @@ When this method is invoked, the user agent MUST execute the following algorithm
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
             |authenticator| and [=set/remove=] |authenticator| from |issuedRequests|.
 
+        :   If the user exercises a user-interface option to cancel the process,
+        ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
+            and [=set/remove=] |authenticator| from |issuedRequests|. Return a {{DOMException}} whose name is "{{NotAllowedError}}".
+
         :   If the {{CredentialRequestOptions/signal}} member is [=present=] and the [=AbortSignal/aborted flag=] is set to
             [TRUE],
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
             and [=set/remove=] |authenticator| from |issuedRequests|. Then
             return a {{DOMException}} whose name is "{{AbortError}}" and terminate this algorithm.
 
+        :   If |issuedRequests| is empty, <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> is not empty, and no |authenticator| will become available for any [=public key credentials=] therein,
+        ::  Indicate to the user that the credential could not be found. When the user acknowledges the dialog, or once |lifetimeTimer| expires, return a {{DOMException}} whose name is "{{NotAllowedError}}".
+
+            Note: One way in which the platform may determine that no |authenticator| will become available is by using the <code>{{transports}}</code> members of <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code>. For example, if all credentials only list {{AuthenticatorTransport/internal}}, but all internal |authenticator|s have been tried, then there is no possibility of satisfying the request. Alternatively, all credentials may require transports that the platform does not support.
```

Comment:
suggest:
Note: One way a platform can determine that no |authenticator| will become available is by examining the <code>{{transports}}</code>  members of the present {{PublicKeyCredentialDescriptor}} [=list/items=] of <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code>, if any. For example, if all {{PublicKeyCredentialDescriptor}} [=list/items=] list only  {{AuthenticatorTransport/internal}}, but all internal |authenticator|s have been tried, then there is no possibility of satisfying the request. Alternatively, all {{PublicKeyCredentialDescriptor}} [=list/items=] may list <code>{{transports}}</code> that the platform does not support.

Reply:
Done.